### DISCUSSION TITLE
Support custom date format or date only for `lastUpdated`

### DISCUSSION THREAD

**Discussion by narumi147** [vuepress/core#1394] [Ideas]
Raised 2y ago, but no response.
- https://github.com/vuepress/vuepress-next/issues/476

https://github.com/vuepress/vuepress-next/blob/db3ce2cc5fcee113cdd16d84c89d08fba2f6929e/ecosystem/theme-default/src/client/components/PageMeta.vue#L69

Support format callback would be pretty nice.

**Comment by Mister-Hope** [comment]
pr welcome

**Comment by narumi147** [comment]
also tried once, but lack js experience (；へ：)
